go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/google/go-cmp v0.3.1
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/projectriff/system v0.0.0-20191129144348-09c47935c6ab
+	github.com/projectriff/system v0.0.0-20191120234226-fe8b3d054ce9
```

Comment:
let's bump this is master after projectriff/system#224 merges

Reply:
of course

Reply:
That's the point of the TODO in the WIP commit log.